<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Advanced</class>
 <widget class="QDialog" name="Advanced">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>312</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="fraDSP">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>291</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>DSP</string>
   </property>
   <widget class="QLabel" name="lblLTtrim">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>LT fine-tune</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbTRPguard">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbBLRrate">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblPURguard">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>PUR guard</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblTRPguard">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TRP guard</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="nmrLTtrim">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbPURguard">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblBLRrate">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>BLR rate</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblAnalogPZ">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Analog P/Z</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="nmrAnalogPZ">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>140</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="fraGPIO">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>291</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>GPIO</string>
   </property>
   <widget class="QCheckBox" name="chkExtAcqCtlEnable">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Enable external acquisition control</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblGPIO1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>GPIO1 function</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbGPIO1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbGPIO1neg">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Neg</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbGPIO1pos">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Pos</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbGPIO2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblGPIO2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>GPIO2 function</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbGPIO2pos">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Pos</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbGPIO2neg">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>140</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Neg</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="cmdApply">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>360</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdClose">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>360</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>nmrLTtrim</tabstop>
  <tabstop>cmbBLRrate</tabstop>
  <tabstop>cmbPURguard</tabstop>
  <tabstop>cmbTRPguard</tabstop>
  <tabstop>nmrAnalogPZ</tabstop>
  <tabstop>chkExtAcqCtlEnable</tabstop>
  <tabstop>cmbGPIO1</tabstop>
  <tabstop>rbGPIO1pos</tabstop>
  <tabstop>rbGPIO1neg</tabstop>
  <tabstop>cmbGPIO2</tabstop>
  <tabstop>rbGPIO2pos</tabstop>
  <tabstop>rbGPIO2neg</tabstop>
  <tabstop>cmdApply</tabstop>
  <tabstop>cmdClose</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
